### ISSUE TITLE
BallotNav: Update alt text for the BallotNav image to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3874]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `BallotNav` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [ ] Change the image alt property value within `_projects/ballot-nav.md`:

**From:**
```
alt: "Two hands holding a mobile device with BallotNav's web application on display."
```

**To:**
```
alt: "Ballot Nav"
```

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move the epic issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- https://www.hackforla.org/projects/ballot-nav

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/ballot-nav.md`
- https://github.com/hackforla/website/blob/gh-pages/_projects/ballot-nav.md
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by adevlinb** [comment]
Availability for this week: Monday - Friday 5:00pm - 6:00pm PT.
My estimated ETA for completing this issue: 20 Minutes - by Monday 6:00PM PT.

**Comment by adevlinb** [comment]
Provide Update
1. Progress - completed... working to create pull request currently
2. Blockers - none / working through pull request
3. Availability - M-F / 12PM - 1PM && 5PM - 6PM PT 
4. ETA -> By Monday March, 20 6PMPT
